examples/versions.tf

```diff
@@ -1,12 +1,13 @@
-
 terraform {
   required_version = ">= 1.2.5"
   required_providers {
     aws = {
-      source = "hashicorp/aws"
+      source  = "hashicorp/aws"
+      version = ">= 4.0.0"
```

Comment:
Below we used to require `>= 4.27.0`, is there a reason for `>=4.0.0` over that version?

Reply:
As discussed in https://github.com/ministryofjustice/cloud-platform-terraform-sqs/pull/51#discussion_r1285877561.